### DISCUSSION TITLE
How to use container-fluid

### DISCUSSION THREAD

**Discussion by cephaspad** [abpframework/abp#16310] [Q&A]
I use **Abp Framework 7.1.1**, and **Blazor-Server**.

How can I make **container-fluid** instead of **container** as default div container for UI?

**Comment by muzkaviy** [comment]
hi,
please check "Copy & Customize" section in this url:
https://docs.abp.io/en/abp/latest/UI/Blazor/Basic-Theme?UI=BlazorServer

after you download the source code, you can change

`<div class="container"> -----> <div class="container-fluid">` in
/packages/Volo.Abp.AspNetCore.Components.Web.BasicTheme/Themes/Basic/MainLayout.razor
